app/models/emoji.rb

```diff
@@ -48,19 +43,30 @@ def self.exists?(name)
   end
 
   def self.[](name)
-    Emoji.custom.detect { |e| e.name == name }
+    name = name.delete_prefix(':').delete_suffix(':')
```

Comment:
I wonder if this might be faster 
```suggestion
    name = name[1...-1]
```